packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission-for-mt5.jsx

```diff
@@ -30,7 +30,7 @@ const POISubmissionForMT5 = ({
                 is_idv_supported &&
                 Number(idv_submissions_left) > 0 &&
                 !is_idv_disallowed &&
-                jurisdiction_selected_shortcode !== 'vanuatu'
+                jurisdiction_selected_shortcode !== Jurisdiction.VANUATU
```

Comment:
You have 25.5% duplication. I suggest you create an improvement card to fix this issue later. Other than that everything LGTM.